crates/delta/src/routes/webhooks/webhook_fetch.rs

```diff
@@ -0,0 +1,28 @@
+use revolt_quark::{Db, Ref, Result, models::{Webhook, File}};
+use rocket::serde::json::Json;
+use serde::{Serialize, Deserialize};
+
+
+// This route is used to get the info about the webhook by clients to get the name and avatar,
+// so this function cant return the token or require any permissions.
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+pub struct WebhookData {
+    #[serde(rename = "_id")]
+    pub id: String,
+    pub name: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub avatar: Option<File>,
+    pub channel: String,
+}
+
+/// # gets a webhook
```

Comment:
All the route titles should be capitalised and the description should start with a capital (this shows up on Swagger and webhooks would break the convention)